### PR TITLE
Feature/sg 143 document commands for all recipes

### PR DESCRIPTION
**Pull request by Louis-Dupont**

### CODE REVIEW

File: src/super_gradients/recipes/training_hyperparams/cifar10_resnet_train_params.yaml [hunk 1]
```diff
@@ -2,7 +2,13 @@ defaults:
   - default_train_params
 
 max_epochs: 250
-lr_updates: [100, 150, 200]
+
+lr_updates:
```
Comment (ofrimasad) [inline src/super_gradients/recipes/training_hyperparams/cifar10_resnet_train_params.yaml:6]: hydra actually has a builtin function for that:
```lr_updates : ${range:100,200,50}```

File: src/super_gradients/recipes/training_hyperparams/imagenet_mobilenetv2_train_params.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ max_epochs: 450
 lr_mode: step
 initial_lr: 0.032   # for total batch-size of 512
 lr_decay_factor: 0.973
-lr_updates:
+lr_updates: #FIXME: This creates a bug because a list is expected and it gets a np array
```
Comment (ofrimasad) [inline src/super_gradients/recipes/training_hyperparams/imagenet_mobilenetv2_train_params.yaml:9]: hydra actually has a builtin function for that:
```lr_updates : ${range:100,200,50}```

**Review by ofrimasad — COMMENTED** [review]
looks very nice

. some minor comments

File: src/super_gradients/recipes/training_hyperparams/cifar10_resnet_train_params.yaml (hunk 1 above)
Comment (Louis-Dupont) [reply src/super_gradients/recipes/training_hyperparams/cifar10_resnet_train_params.yaml:6]: It's actually not builtin, it requires to add :
`OmegaConf.register_new_resolver("range", lambda *args: np.arange(*args).tolist())`

File: src/super_gradients/recipes/training_hyperparams/imagenet_mobilenetv2_train_params.yaml (hunk 2 above)
Comment (Louis-Dupont) [reply src/super_gradients/recipes/training_hyperparams/imagenet_mobilenetv2_train_params.yaml:9]: It's actually not builtin, it requires to add :
`OmegaConf.register_new_resolver("range", lambda *args: np.arange(*args).tolist())`

Comment (ofrimasad) [reply src/super_gradients/recipes/training_hyperparams/imagenet_mobilenetv2_train_params.yaml:9]: ok. so keep it as it is

**Review by ofrimasad — DISMISSED** [review]
LGTM

**Review by shaydeci — CHANGES_REQUESTED** [review]
One minor comment then lets merge...

File: README.md [hunk 3]
```diff
@@ -449,7 +450,196 @@ Devices[https://arxiv.org/pdf/1807.11164](https://arxiv.org/pdf/1807.11164)
...
+
+## Training Recipes
+
+We defined recipes to ensure that anyone can reproduce our results in the most simple way.
+
+
+**How to run a recipe**
+
+The recipes are defined in .yaml format and we use the hydra library to allow you to easily customize the parameters.
+The basic syntax is as follow:
```
[the remark this answers is not in the document]
Comment (Louis-Dupont) [reply README.md]: Updated in 368c4fd3eb1e941413bd92f75cdea072bf00829d

[the remark this answers is not in the document]
Comment (shaydeci) [reply README.md]: Does this work for you ? Don't you need export PYTHONPATH=/super_gradients/src:super_gradients/ ?

File: README.md [hunk 4]
```diff
@@ -449,7 +450,218 @@ Devices[https://arxiv.org/pdf/1807.11164](https://arxiv.org/pdf/1807.11164)
...
+
+
+**How to run a recipe**
+
+The recipes are defined in .yaml format and we use the hydra library to allow you to easily customize the parameters.
+The basic basic syntax is as follow:
+```
+python src/super_gradients/examples/train_from_recipe_example/train_from_recipe.py --config-name=<CONFIG-NAME> dataset_params.data_dir=<PATH-TO-DATASET>
+```
+But in most cases you will want to train on multiple GPU using this syntax:
```
Comment (shaydeci) [inline README.md:484]: multiple GPU's *

**Review by shaydeci — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Louis-Dupont) [reply README.md]: changed in 39e462409ea6ad24070e3764025a1b273051e793

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Louis-Dupont) [reply README.md]: Yes, changed in 39e462409ea6ad24070e3764025a1b273051e793

**Pull request merged**